fetch/fetch.go

```diff
@@ -0,0 +1,174 @@
+package fetch
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/ssm"
+)
+
+var (
+	client *ssm.SSM
+	paths  []string
+	tags   []string
+
+	trueBool = true
+)
+
+func init() {
+	// initialize AWS client
+	initClient()
+}
+
+func initClient() {
+	session := session.Must(session.NewSession())
+	client = ssm.New(session)
+}
+
+func FetchParams(paths, tags []string) ([]*ssm.Parameter, error) {
```

Comment:
I'm assuming there were no actual code changes when moving these from the other file?

Reply:
ah, there were. i'll make sure there's a commit for that.

Reply:
actually the first commit shows those changes